src/main/java/use_cases/GameCreationInteractor.java

```diff
@@ -0,0 +1,177 @@
+ * to create a Game, and sends a response model to the controller on whether a Game was created or not.
+ */
+public class GameCreationInteractor implements GameCreationInputBoundary {
+
+    private final DataAccess dataAccess;
+    private Game createdGame;
+    private GameState newGameState;
+    public GameCreationInteractor(DataAccess dataAccess) {
+        this.dataAccess = dataAccess;
+    }
+
+    /**
+     * Created a response model, and Game if the request is valid.
+     * @param request a GameCreationRequestModel containing details from the User.
+     * @return a GameCreationResponseModel containing whether a Game was created (if User input was valid or not).
+     */
+    @Override
+    public GameCreationResponseModel createGameResponse(GameCreationRequestModel request) {
+        if(!(validAmountPlayers(request))) {
+            return new GameCreationResponseModel(false);
+        }
+        Deck deck = buildDeck();
+        ArrayList<Player> players = buildPlayers(request, deck);
+        Game game = buildGame(players, deck);
+        createdGame = game;
+        return new GameCreationResponseModel(true);
+    }
+
+    /**
+     * Get the Game this interactor has created.
+     * @return the Game instance that has been created by this class.
+     */
+    public Game getCreatedGame() {
+        return createdGame;
+    }
+
+    /**
+     * Get the GameState created by this class.
+     * @return a GameState instance.
+     */
+    public GameState getNewGameState() {
+        return newGameState;
+    }
+
+    /**
+     * Check if the requested Players to create has enough players and enough HumanPlayers to create a Game.
+     * @param request The request containing details from the User regarding types of Players to create.
+     * @return True iff more than 2 Players total and at least 1 HumanPlayer requested.
+     */
+    private boolean validAmountPlayers(GameCreationRequestModel request) {
+        // check there are at least 2 players in the Game
+        if(request.getPlayersInfo().keySet().size() < 2) {
+            return false;
+        }
+        int numHumanPlayer = 0;
+        for (String name: request.getPlayersInfo().keySet()) {
+            if(!(request.isComputer(name))) {
+             numHumanPlayer ++;
+            }
+        }
+        // The request contains at least 1 human player
+        return numHumanPlayer >= 1;
+    }
+
+    /**
+     * Build the Deck instance that will be used by the Game that is to be built.
+     * @return a Deck instance.
+     */
+    private Deck buildDeck() {
+        return new StandardDeck();
+    }
+
+    /**
+     * Build a Hand instance for a Player in this Game.
+     * @param deck The Deck which this Hand's Cards will be from.
+     * @return a Hand instance representing the Hand of a Player in the Game to be built.
+     */
+    private Hand buildHand(Deck deck) {
+        Hand hand = new Hand(new ArrayList<>());
+        int NUMBERCARDS = 5;
```

Comment:
Consider extracting this magic number out of this method and initialize it as an instance attribute.

Reply:
replaced this with a constant instead.